install_scripts.sh

```diff
@@ -102,6 +102,16 @@ mc_files=(
 )
 copy_files_by_list ${TOPDIR}/init/Magic_Castle ${INSTALL_PREFIX}/init/Magic_Castle "${mc_files[@]}"
 
+# Copy for init/modules/EESSI directory
+mc_files=(
+   2023.06.lua
+)
+if [ -n "$EESSI_CVMFS_REPO" ]; then
+    copy_files_by_list ${TOPDIR}/init/modules/EESSI ${EESSI_CVMFS_REPO}/modules/EESSI "${mc_files[@]}"
+else
+    echo "Error: EESSI_CVMFS_REPO is required for copying the module files." >&2
+fi
```

Comment:
This adds a new dependency on an environment variable. I'd suggest to add a new argument to the script, say, `--cvmfs-repo` or `--cvmfs-repo-path`, which can then be used in a similar fashion to `--prefix`.

Reply:
I will wait for a final decision on where to place the module and act accordingly

Reply:
Since this is not a trivial change, I think we go with the original approach for now. This only really becomes relevant when there are a few version options (and a symlink would solve that)